.goreleaser.template.yml

```
@@ -107,3 +107,14 @@ release:
   disable: $DISABLE_RELEASE_PIPELINE
   extra_files:
   - glob: 'rukpak.yaml'
+  header: |
+    ## Installation
+
+    ```bash
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
```

Does it make sense to export the `CERT_MGR_VERSION` we're using in the `Makefile` to the environment and reference it here?

```suggestion
    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/{{ .Env.CERT_MGR_VERSION }}/cert-manager.yaml
```

---

Nice catch. I'll play around with this in my fork real quick and push up some changes if I can get it working.